src/components/select-field/MultiSelectField.js

```diff
@@ -8,6 +8,20 @@ type Props = {
     onChange: Function,
 };
 
-const MultiSelectField = ({ ...rest }: Props) => <BaseSelectField {...rest} multiple />;
+const optionsWithClearOption = (options: Array<Object>, shouldShowClearOption: boolean) => {
+    if (shouldShowClearOption) {
+        const updatedOptions = options;
+        updatedOptions.unshift({
+            value: 'clear',
+            displayText: 'Clear All',
```

Comment:
Don't we need to localize `Clear All`? Make sure to create `messages.js` and add `Clear All` string there so it can be translated. You should be using something like `<FormattedMessage {...messages.clearAllButton} />` instead of hard-coded string literal.